Mathlib/RingTheory/TensorProduct/MvPolynomial.lean

```diff
@@ -186,6 +186,43 @@ noncomputable def scalarRTensorAlgEquiv :
     MvPolynomial σ R ⊗[R] N ≃ₐ[R] MvPolynomial σ N :=
   rTensorAlgEquiv.trans (mapAlgEquiv σ (Algebra.TensorProduct.lid R N))
 
+variable (A : Type*) [CommRing A] [Algebra R A]
+
+/-- Tensoring `MvPolynomial σ R` on the left by an `R`-algebra `A` is algebraically
+equivalent to `M̀vPolynomial σ A`. -/
+noncomputable def algebraTensorAlgEquiv :
+    A ⊗[R] MvPolynomial σ R ≃ₐ[A] MvPolynomial σ A := AlgEquiv.ofAlgHom
+  (Algebra.TensorProduct.lift
+    (Algebra.ofId A (MvPolynomial σ A))
+    (MvPolynomial.mapAlgHom <| Algebra.ofId R A) (fun _ _ ↦ Commute.all _ _))
+  (aeval (fun s ↦ 1 ⊗ₜ X s))
+  (by ext s; simp)
+  (by ext s; simp)
+
+@[simp]
+lemma algebraTensorAlgEquiv_tmul (a : A) (p : MvPolynomial σ R) :
+    algebraTensorAlgEquiv (R := R) A (a ⊗ₜ p) = a • MvPolynomial.map (algebraMap R A) p := by
```

Comment:
Is the `(R := R)` necessary here? If it is, I don't see how it could be implicit elsewhere and I'd suggest we make it explicit in the first place.

Reply:
In this example it is indeed not necessary, in the `symm_X` lemma below it is necessary though. Should I make it explicit anyway?

Reply:
In that case I think we should make it explicit.

Reply:
Done.